.github/workflows/rust-test-with-style.yml

```diff
@@ -18,19 +18,19 @@ jobs:
     - name: Rust setup
       uses: dtolnay/rust-toolchain@master
       with:
-        # Note: nightly required for coverage of Doctests with tarpaulin
+        # Note: nightly required for coverage of Doctests
         toolchain: nightly
+    - name: Install cargo-llvm-cov
+      uses: taiki-e/install-action@cargo-llvm-cov
     - name: Rust test with coverage
-      uses: actions-rs/tarpaulin@v0.1
       env:
         CC: ${{ matrix.compiler }}
         FC: gfortran-11
-      with:
-        args: '--run-types Doctests Tests --exclude-files backends/* gallery/* include/* interface/* --out Xml'
+      run: cargo llvm-cov test --doctests --lcov --output-path lcov.info
```

Comment:
Will this still exclude the libCEED C files from the coverage?

Reply:
Yeah, it excludes them, but there is a way to include FFI coverage (that I'm not activating now, but perhaps we would want in the future).

Reply:
The coverage in the Rust files looks a lot more accurate now

Reply:
Huh, I wouldn't have expected this much change from using `llvm-cov`, but I'll take it.

https://app.codecov.io/gh/CEED/libCEED/pull/1120

![image](https://user-images.githubusercontent.com/3303/208787227-1e6fb65a-368e-40d6-8941-e82aa4c6cec2.png)

Reply:
Its because tarpaulin incorrectly marked a ton of lines as 'uncovered' when one function call split across multiple lines